api/validators/GenericRequests.ts

```diff
@@ -8,3 +8,18 @@ export class Pagination implements IPagination {
   @Min(0)
   limit?: number;
 }
+
+export class EmailParam {
+  @IsEmail()
+  email:string;
+}
```

Comment:
nit: leave a space between the key and value in an object for consistency and readability: `email: string;` vs. `email:string`

Reply:
I'm surprised this wasn't caught by the linter.